### PR TITLE
Fixed DIIID CO2 geo with OMFITprofiles

### PR DESCRIPTION
**Pull request by AreWeDreaming**
The DIII-D mapping used arbitrary endpoints for the CO2 interferometer. The machine mapping was updated with the coordinates from OMFITprofiles.

### CODE REVIEW

File: omas/machine_mappings/d3d.py [hunk 1]
```diff
@@ -523,15 +523,16 @@ def interferometer_hardware(ods, pulse):
     ods['interferometer.channel.0.identifier'] = ods['interferometer.channel.0.name'] = 'r0'
     r0 = ods['interferometer.channel.0.line_of_sight']
     r0['first_point.phi'] = r0['second_point.phi'] = 225 * (-np.pi / 180.0)
-    r0['first_point.r'], r0['second_point.r'] = 3.0, 0.8  # These are not the real endpoints
```
Comment (orso82) [inline omas/machine_mappings/d3d.py:526]: @AreWeDreaming you may want to remove the comment above: "The chord endpoints ARE NOT RIGHT. Only the R for vertical lines or Z for horizontal lines is right.".

**Comment by AreWeDreaming** [comment]
I will. I also just noticed that the CO2 is double pass. Per IMAS convention it should [have 3 points](https://gafusion.github.io/omas/schema/schema_interferometer.html).

**Comment by orso82** [comment]
About the DIII-D interferometer being double pass, is then the density reported in MDS+ already halved? What's the convention in IMAS for that?

**Comment by AreWeDreaming** [comment]
"Line integrated density, possibly obtained by a combination of multiple interferometry wavelengths. Corresponds to the density integrated along the full line-of-sight (i.e. forward AND return for a reflected channel: NO dividing by 2 correction)"

**Comment by AreWeDreaming** [comment]
I have been trying to get TS and CO2 to agree in my IDA workflow for ages. They don't and a factor of 2 won't make things much better.

**Comment by AreWeDreaming** [comment]
Also from the CO2 wiki entry: 
All signals are for the double pass line-integrated density along a given chord in (m/cm3) ≡ (106 m-2): denr0

**Comment by AreWeDreaming** [comment]
Nvm. the third point was always there, I just didn't see it =/

**Pull request merged**